Formula/git-annex.rb

```diff
@@ -40,6 +40,30 @@ def install
     bin.install_symlink "git-annex" => "git-annex-shell"
   end
 
+  plist_options :startup => true, :manual => "git annex assistant --autostart"
```

Comment:
does it really need root privileges? If not the `:startup => true` should go

Reply:
I may not fully understand the implications of these options.  I judged off of:
https://www.rubydoc.info/github/Homebrew/brew/Formula.plist_options

The rational being that the service should auto-start on start-up, not wait for the user to manually bring it up each time they boot.

Reply:
It'll load when the user logs. Note that `httpd` and `php` formulae both use `plist_options :manual`